Controller/ConnectController.php

```diff
@@ -157,11 +109,7 @@ public function registrationAction(Request $request, $key)
         ;
 
         /* @var $form FormInterface */
-        if ($this->container->getParameter('hwi_oauth.fosub_enabled')) {
-            $form = $this->container->get('hwi_oauth.registration.form.factory')->createForm();
-        } else {
-            $form = $this->container->get('hwi_oauth.registration.form');
-        }
+        $form = $this->container->get('hwi_oauth.registration.form.factory')->createForm();
```

Comment:
This should be reverted probably, no? Cause case is not same.

Reply:
I forgot to mention this, I couldn't find a place where the `hwi_oauth.registration.form` service is registered. It looks like it was remove here https://github.com/hwi/HWIOAuthBundle/commit/f4fde1cfc761ed130140756e435913c78431ead2#diff-ebba42a22ab93908a46d40ab71b2d60a

Reply:
Ah true, so forgotten leftover.